template/.eslintrc.js

```diff
@@ -44,6 +44,8 @@ module.exports = {
       'js': 'never',
       'vue': 'never'
     }],
+    // allow vuex
+    'no-param-reassign': ["error", { 'props': false }],
```

Comment:
it is not a good practice, as it disable checking the rule to all files.
a better way is disabling rules with inline comments.